workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@backstage-community/plugin-multi-source-security-viewer",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "pluginId": "multi-source-security-viewer",
+    "pluginPackages": [
+      "@backstage-community/plugin-multi-source-security-viewer",
+      "@backstage-community/plugin-multi-source-security-viewer-common"
+    ]
+  },
+  "sideEffects": [
```

Comment:
Only a small number of the plugins do this, can you clarify the need, please?

Reply:
I don't need it. Removed.